kratos/python/add_vector_to_python.cpp

```diff
@@ -216,6 +216,10 @@ namespace Python
                                     tmp[i] = py::cast<double>(input[i]);
                                 return tmp;
                                 }));
+        vector_binder.def(py::init([](const VectorSlice& input){
+                                Vector tmp(input);
+                                return tmp;
+                                }));
```

Comment:
```suggestion
        vector_binder.def(py::init([](const VectorSlice& input) -> Vector {
                                return input;
                                }));
```

this way you avoid one copy by using the implicit conversion